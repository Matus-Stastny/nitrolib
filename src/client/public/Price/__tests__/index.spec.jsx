/* @flow */
import * as React from "react";
import { shallow } from "enzyme";

import WrappedPrice from "../index";

const Price = WrappedPrice.WrappedComponent;

describe("#Price", () => {
  it("should render correctly", () => {
    const wrapper = shallow(<Price value={1234} currency="EUR" locale="en" translate={id => id} />);

    expect(wrapper).toMatchSnapshot();
  });
});
