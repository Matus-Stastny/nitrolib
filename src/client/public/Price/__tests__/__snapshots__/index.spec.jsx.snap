// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#Price should render correctly 1`] = `"â‚¬1,234.00"`;
